### PR TITLE
Fix asciiToString

### LINKED ISSUE #261: Fix asciiToString method
There is currently a method called `asciiToString` that lives in `/lib/asciiToString.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test asciiToString` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by domosapien**
* Adds distance skeleton

Closes #261

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/distance.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const { distance } = require("../lib");
+
+describe("ASCIIToString", () => {
```
Comment (BlakeGuilloud) [inline test/distance.test.js]: Cool! This is a typo though. Should be describe("distance"

Comment (domosapien) [reply test/distance.test.js]: Argh, missed my copy and paste
